### DISCUSSION TITLE
NodeMCU pinouts & image

### DISCUSSION THREAD

**Discussion by LionBS76** [softwarecrash/Solar2MQTT-ESP8266#142] [Q&A] [answered]
Hello!
What image of three available ( d1 mini, esp-01, esp-12 ) should I use for NodeMCU board ?
I've choosen esp-12. Is it correct ?
And what pinouts, like D1 Mini ?

**Comment by LionBS76** [comment]
Answer on my own question. :)
Image esp-12 doesn't work maybe because wrong pinouts used.
Image D1 Mini with standard pinouts (as wirng diagram) works!

**Reply by softwarecrash** [reply, marked as answer]
the esp-12 image have different pinout special for the selled wifi dongles that can be reflashed.
the wemos image is the right, its the same chip as nodemcu, and the most pinout following the wemos schematics
